src/gui/mainwindow.cpp

```diff
@@ -843,6 +843,11 @@ void MainWindow::createKeyboardShortcuts()
     m_ui->actionDelete->setShortcutContext(Qt::WidgetShortcut);  // nullify its effect: delete key event is handled by respective widgets, not here
     m_ui->actionDownloadFromURL->setShortcut(Qt::CTRL + Qt::SHIFT + Qt::Key_O);
     m_ui->actionExit->setShortcut(Qt::CTRL + Qt::Key_Q);
+#ifdef Q_OS_MAC
+    m_ui->actionCloseWindow->setShortcut(QKeySequence::Close);
+#else
+    m_ui->actionCloseWindow->setVisible(false);
```

Comment:
Maybe delete it instead?

Reply:
@glassez I'd expect this functionality to appear on other operating systems in reasonable future, because it is, well, common sense, from what I am aware of. While I definitely can put removeAction here, is there any good reason to change the code that is a temporary anyway, since I simply cannot test it on other OS?